launch/sample_node.launch

```diff
@@ -2,37 +2,60 @@
 
 <launch>
```

Comment:
as this is now being forked from the official repo with the idea to finally play back the changes, we might not want do modify this file as possibly others using the original repo rely on it unchanged!
We should put the cob4-adjusted launch file somewhere different...